golangcilint/deps.bzl

```diff
@@ -29,8 +29,8 @@ def _golangcilint_download_impl(ctx):
     if arch not in _CHECKSUMS:
         fail("Unsupported arch {}".format(arch))
 
-    url = _DOWNLOAD_URI.format(version=_VERSION, arch=arch)
-    prefix = _PREFIX.format(version=_VERSION, arch=arch)
+    url = _DOWNLOAD_URI.format(version = _VERSION, arch = arch)
+    prefix = _PREFIX.format(version = _VERSION, arch = arch)
```

Comment:
Why did the formatting here (32-33) and on 13-18 change?

Reply:
@scottgreenup It looks like this was changed by buildifier.